README.md

```diff
@@ -137,19 +137,18 @@ of the most common ones with non default values. For more info take a look at th
 
 #### File Storage System
 (Used on Kernel, ODK and UI Modules)
-- `DJANGO_REMOTE_STORAGE` (optional): Used to specify a custom [Default file storage system](https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-DEFAULT_FILE_STORAGE). If not provided, defaults to the [Django default storage](https://docs.djangoproject.com/en/2.0/ref/files/storage/#django.core.files.storage.FileSystemStorage) Available options: S3, GCS. More information [here](https://django-storages.readthedocs.io/en/latest/index.html)
-- `REMOTE_STATIC_FILES` (optional): Set to "true" to collect static files to the selected `DJANGO_REMOTE_STORAGE`
+- `DJANGO_STORAGE_BACKEND`: Used to specify a [Default file storage system](https://docs.djangoproject.com/en/1.11/ref/settings/#default-file-storage). Available options: filesystem, s3, gcs. More information [here](https://django-storages.readthedocs.io/en/latest/index.html).
```

Comment:
Is there a default? Or is this mandatory?

Reply:
It's mandatory now -- hence the additions of `filesystem` to all compose files. If you can think of a better way to:
- make sure that we always have either `s3` or `gcs` in production and
- include a default for local development without cluttering the compose files
...I'm all ears.
In other words, the trade-off is: more complex compose configuration, more certainty that prod environments will be configured correctly.

Reply:
Maybe it's worth mentioning it in the README that it's mandatory? I don't know, up to you.

I think that's it's good that there's no default here 👍

Reply:
Added to the README.